cmd/acikubectl/cmd/debug.go

```diff
@@ -368,7 +372,21 @@ func clusterReport(cmd *cobra.Command, args []string) {
 				args: nodeItem.argFunc(systemNamespace, podName,
 					nodeItem.cont, nodeItem.args),
 			})
+
+			//Prepare kubectl cp command for aci-conatiners-host
+			tempName = fmt.Sprintf("hostfiles/node-%s/aci-containers-host/k8s-pod-network/", node.Name)
+			cmds = append(cmds, reportCmdElem{
+				name: tempName,
+				args: []string{"cp", systemNamespace + "/" + podName + ":" +
+					"/usr/local/var/lib/aci-containers/k8s-pod-network", tempName},
+			})
```

Comment:
I don't see that the files under k8s-pod-network are being picked up:

noiro@fab14-compute-1:~/anmol/acikctl$ find .
.
./hostfiles
./hostfiles/node-fab14-compute-5
./hostfiles/node-fab14-compute-5/opflex-agent-ovs
./hostfiles/node-fab14-compute-5/aci-containers-host
./hostfiles/node-fab14-compute-5/aci-containers-host/k8s-pod-network
./hostfiles/node-fab14-compute-4
./hostfiles/node-fab14-compute-4/opflex-agent-ovs
./hostfiles/node-fab14-compute-4/aci-containers-host
./hostfiles/node-fab14-compute-4/aci-containers-host/k8s-pod-network
./hostfiles/node-fab14-compute-1
./hostfiles/node-fab14-compute-1/opflex-agent-ovs
./hostfiles/node-fab14-compute-1/aci-containers-host
./hostfiles/node-fab14-compute-1/aci-containers-host/k8s-pod-network

Reply:
As mentioned above, the outer hostfiles dir is empty and only has the dir tree and no files.


```
noiro@fab14-compute-1:~/anmol/acikctl/cluster-report/hostfiles$ pwd
/home/noiro/anmol/acikctl/cluster-report/hostfiles
noiro@fab14-compute-1:~/anmol/acikctl/cluster-report/hostfiles$ find .
.
./node-fab14-compute-5
./node-fab14-compute-5/opflex-agent-ovs
./node-fab14-compute-5/opflex-agent-ovs/faults
./node-fab14-compute-5/opflex-agent-ovs/ids
./node-fab14-compute-5/opflex-agent-ovs/ids/routingDomain.id
./node-fab14-compute-5/opflex-agent-ovs/ids/podsvc.id
./node-fab14-compute-5/opflex-agent-ovs/ids/secGroupSet.id
./node-fab14-compute-5/opflex-agent-ovs/ids/bridgeDomain.id
./node-fab14-compute-5/opflex-agent-ovs/ids/l24classifierRule.id
./node-fab14-compute-5/opflex-agent-ovs/ids/service.id
./node-fab14-compute-5/opflex-agent-ovs/ids/floodDomain.id
./node-fab14-compute-5/opflex-agent-ovs/ids/externalNetwork.id
./node-fab14-compute-5/opflex-agent-ovs/ids/conntrack.id
./node-fab14-compute-5/opflex-agent-ovs/ids/svcstats.id
./node-fab14-compute-5/opflex-agent-ovs/endpoints
./node-fab14-compute-5/opflex-agent-ovs/endpoints/l3out_1_vrf.arp
./node-fab14-compute-5/opflex-agent-ovs/endpoints/aci-containers-default_null-mac.ep
./node-fab14-compute-5/opflex-agent-ovs/services
./node-fab14-compute-5/opflex-agent-ovs/services/e63b24aa-1494-4748-ab0f-b4d162ea1b2e.service
./node-fab14-compute-5/opflex-agent-ovs/services/68ef9013-25b8-40b3-a01c-e3994a7812f2.service
./node-fab14-compute-5/opflex-agent-ovs/droplog
./node-fab14-compute-5/opflex-agent-ovs/droplog/a.droplogcfg
./node-fab14-compute-5/opflex-agent-ovs/snats
./node-fab14-compute-5/opflex-agent-ovs/reboot-conf.d
./node-fab14-compute-5/opflex-agent-ovs/reboot-conf.d/reboot.conf
./node-fab14-compute-5/opflex-agent-ovs/mcast
./node-fab14-compute-5/opflex-agent-ovs/mcast/opflex-groups.json
./node-fab14-compute-5/aci-containers-host
./node-fab14-compute-5/aci-containers-host/k8s-pod-network
./node-fab14-compute-4
./node-fab14-compute-4/opflex-agent-ovs
./node-fab14-compute-4/opflex-agent-ovs/faults
./node-fab14-compute-4/opflex-agent-ovs/ids
./node-fab14-compute-4/opflex-agent-ovs/ids/routingDomain.id
./node-fab14-compute-4/opflex-agent-ovs/ids/podsvc.id
./node-fab14-compute-4/opflex-agent-ovs/ids/secGroupSet.id
./node-fab14-compute-4/opflex-agent-ovs/ids/bridgeDomain.id
./node-fab14-compute-4/opflex-agent-ovs/ids/l24classifierRule.id
./node-fab14-compute-4/opflex-agent-ovs/ids/service.id
./node-fab14-compute-4/opflex-agent-ovs/ids/floodDomain.id
./node-fab14-compute-4/opflex-agent-ovs/ids/externalNetwork.id
./node-fab14-compute-4/opflex-agent-ovs/ids/conntrack.id
./node-fab14-compute-4/opflex-agent-ovs/ids/svcstats.id
./node-fab14-compute-4/opflex-agent-ovs/endpoints
./node-fab14-compute-4/opflex-agent-ovs/endpoints/l3out_1_vrf.arp
./node-fab14-compute-4/opflex-agent-ovs/endpoints/aci-containers-default_null-mac.ep
./node-fab14-compute-4/opflex-agent-ovs/services
./node-fab14-compute-4/opflex-agent-ovs/services/e63b24aa-1494-4748-ab0f-b4d162ea1b2e.service
./node-fab14-compute-4/opflex-agent-ovs/services/68ef9013-25b8-40b3-a01c-e3994a7812f2.service
./node-fab14-compute-4/opflex-agent-ovs/droplog
./node-fab14-compute-4/opflex-agent-ovs/droplog/a.droplogcfg
./node-fab14-compute-4/opflex-agent-ovs/snats
./node-fab14-compute-4/opflex-agent-ovs/reboot-conf.d
./node-fab14-compute-4/opflex-agent-ovs/reboot-conf.d/reboot.conf
./node-fab14-compute-4/opflex-agent-ovs/mcast
./node-fab14-compute-4/opflex-agent-ovs/mcast/opflex-groups.json
./node-fab14-compute-4/aci-containers-host
./node-fab14-compute-4/aci-containers-host/k8s-pod-network
./node-fab14-compute-1
./node-fab14-compute-1/opflex-agent-ovs
./node-fab14-compute-1/opflex-agent-ovs/faults
./node-fab14-compute-1/opflex-agent-ovs/ids
./node-fab14-compute-1/opflex-agent-ovs/ids/routingDomain.id
./node-fab14-compute-1/opflex-agent-ovs/ids/podsvc.id
./node-fab14-compute-1/opflex-agent-ovs/ids/secGroupSet.id
./node-fab14-compute-1/opflex-agent-ovs/ids/bridgeDomain.id
./node-fab14-compute-1/opflex-agent-ovs/ids/l24classifierRule.id
./node-fab14-compute-1/opflex-agent-ovs/ids/service.id
./node-fab14-compute-1/opflex-agent-ovs/ids/floodDomain.id
./node-fab14-compute-1/opflex-agent-ovs/ids/externalNetwork.id
./node-fab14-compute-1/opflex-agent-ovs/ids/conntrack.id
./node-fab14-compute-1/opflex-agent-ovs/ids/svcstats.id
./node-fab14-compute-1/opflex-agent-ovs/endpoints
./node-fab14-compute-1/opflex-agent-ovs/endpoints/aci-containers-default_null-mac.ep
./node-fab14-compute-1/opflex-agent-ovs/endpoints/ebd3c90e-112e-409d-b4ae-3abfca8d7e84_2e9a9b048778c7493b06d5054be2ba209262863aecdd835d3914dbd3b1f9b454_veth17ae1baa.ep
./node-fab14-compute-1/opflex-agent-ovs/endpoints/d92c027b-3093-4d8c-84c3-7650b4fc4761_d1aac99f184696d0c3a31d731fcafbcc32f94e58bc56606b252b3ed1781f87c6_vetheb2ad45f.ep
./node-fab14-compute-1/opflex-agent-ovs/services
./node-fab14-compute-1/opflex-agent-ovs/services/e63b24aa-1494-4748-ab0f-b4d162ea1b2e.service
./node-fab14-compute-1/opflex-agent-ovs/services/68ef9013-25b8-40b3-a01c-e3994a7812f2.service
./node-fab14-compute-1/opflex-agent-ovs/droplog
./node-fab14-compute-1/opflex-agent-ovs/droplog/a.droplogcfg
./node-fab14-compute-1/opflex-agent-ovs/snats
./node-fab14-compute-1/opflex-agent-ovs/reboot-conf.d
./node-fab14-compute-1/opflex-agent-ovs/reboot-conf.d/reboot.conf
./node-fab14-compute-1/opflex-agent-ovs/mcast
./node-fab14-compute-1/opflex-agent-ovs/mcast/opflex-groups.json
./node-fab14-compute-1/aci-containers-host
./node-fab14-compute-1/aci-containers-host/k8s-pod-network
./node-fab14-compute-1/aci-containers-host/k8s-pod-network/2e9a9b048778c7493b06d5054be2ba209262863aecdd835d3914dbd3b1f9b454
./node-fab14-compute-1/aci-containers-host/k8s-pod-network/d1aac99f184696d0c3a31d731fcafbcc32f94e58bc56606b252b3ed1781f87c6
```